jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -44,11 +44,14 @@ The instructions below will clone the repository containing the Server v2.19.x t
 The migration script will:
 
 * Stop your v2.19.x application
-* Create a tarball of your v2.19.x application's PostgreSQL and Mongo databases
-* Archive existing application data for Vault and CircleCI encryption/signing keys
-* Export the tarball to your v3.x installation (exported data stores are stored in `migrate/circleci_export`, which may be useful for debugging purposes).
-* Scale v3.x application deployments down to zero, and then to one
-* Import the data from the previously exported tarball to your new v3.x instance
+* Perform pre-flight checks to confirm namespace and datastores for 2.19.x. 
+* Create a tarball of your v2.19.x application's PostgreSQL and Mongo databases.
+* Archive existing application data for Vault and CircleCI encryption/signing keys.
+* Export the 2.19.x tarball to your v3.x installation (exported data stores are stored in `migrate/circleci_export`, which may be useful for debugging purposes).
```

Comment:
We should call out here we are actually copying the files locally before initiating the 3.x import.

Reply:
Also `migrate/circleci_export` is slightly incorrect - the `circleci_export` directory will always be in the current working directory. If a user were to `cd /tmp` before running the migration script, the export would end up in `/tmp/circleci_export`.